chat_test.go

```diff
@@ -302,18 +302,36 @@ func TestGetEmoteSets(t *testing.T) {
 		statusCode         int
 		options            *Options
 		GetEmoteSetsParams *GetEmoteSetsParams
+		expectedEmotes     []EmoteWithOwner
 		respBody           string
 	}{
 		{
 			http.StatusBadRequest,
 			&Options{ClientID: "my-client-id"},
 			&GetEmoteSetsParams{EmoteSetIDs: nil},
+			nil,
 			`{"error":"Bad Request","status":400,"message":"The parameter \"emote_set_id\" was malformed: the value must be greater than or equal to 1"}`,
 		},
 		{
 			http.StatusOK,
 			&Options{ClientID: "my-client-id"},
 			&GetEmoteSetsParams{EmoteSetIDs: []string{"300678379"}},
+			[]EmoteWithOwner{
+				{
+					Emote: Emote{
+						ID:   "301147694",
+						Name: "sixone3SixDab",
+						Images: EmoteImage{
+							Url1x: "https://static-cdn.jtvnw.net/emoticons/v1/301147694/1.0",
+							Url2x: "https://static-cdn.jtvnw.net/emoticons/v1/301147694/2.0",
+							Url4x: "https://static-cdn.jtvnw.net/emoticons/v1//3.0",
```

Comment:
No idea why is there no emote ID here. Would have to adjust JSON in line 335 as well.

```suggestion
							Url4x: "https://static-cdn.jtvnw.net/emoticons/v1/301147694/3.0",
```